src/App.js

```diff
@@ -1,3 +1,6 @@
-const heading2 = React.createElement("h2",{},"Hello React");
+import React from "react";
+import ReactDOM from "react-dom/client";
```

Comment:
can give an empty line between direct imports and file imports for better readability